java/mykola-melnyk/src/com/zoolatech/lecture6/tasks/_1/Task1.java

```diff
@@ -0,0 +1,78 @@
+package com.zoolatech.lecture6.tasks._1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Class City has the following properties: string name, string country, int population.
+ * Create methods (all of them need to use Stream API):
+ * a) that accepts a list of cities and a value of a population,
+ * and returns true if all cities in the list have the population bigger than a passed value
+ * b) that accepts a list of cities and a country name and returns a city with the biggest
+ * population for that country
+ * c) that accepts a list of cities and returns a map, where the key is the city name and
+ * a value - is a corresponding city object (note: the list might contain duplicates).
+ */
+
+public class Task1 {
+
+    private static boolean checkPopulationIsBigger(ArrayList<City> cityList, int population) {
+        long populationCount = cityList.stream()
```

Comment:
try to simplify this code using the Stream's method `allMatch`

Reply:
Done.